### DISCUSSION TITLE
Run remotely-save silently

### DISCUSSION THREAD

**Discussion by SamiChab** [remotely-save/remotely-save#242] [General]
Hello. Thank you for the amazing work on this plugin !

The synchronization that is started automatically at startup runs silently (no toast is displayed on the screen). I wonder if it is also possible to run remotely-save silently when performing a manual sync. Of course if an error happen, it would be great to show a toast (or even an alert modal)

![Screenshot_20221109-020610](https://user-images.githubusercontent.com/28505397/200711854-3bcd6aa8-4fe0-48a8-9047-ef67e5985e2b.png)

**Comment by fyears** [comment]
in the latest version the notices are reduced to eliminate the distractions. but i am going to keep that notices because users should know the result if they trigger the sync manually

**Reply by SamiChab** [reply]
It's way better now. Thank you !!
